index.js

```diff
@@ -0,0 +1,53 @@
+import addBook from './modules/addBook.js';
+import { DateTime } from './modules/luxon.js';
+
+const booksForm = document.querySelector('.books-form');
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+
+window.onload = () => {
+  const date = document.getElementById('date');
+  date.innerHTML = DateTime.now().toLocaleString(DateTime.DATETIME_HUGE);
+};
```

Comment:
- [ ] Kindly put the date under the navigation bar not inside the navigation to be consistent with wireframe guidelines.
![image](https://user-images.githubusercontent.com/53402600/183913083-b793d255-c9d7-4be3-bb84-b4eecd39a66a.png)

Reply:
@omarsalem7 Oh! Thank you for spotting that for me. Thank you for the review. I will fix that now.